### ISSUE TITLE
ssh config file from generate_ssh_keys.yaml broken

### ISSUE THREAD

**Issue by therevoman** [RedHatOfficial/ocp4-helpernode#69]
While testing this on a new cluster, I had some systems I wanted to access via IP..  The new ~/.ssh/config file has the line
`Host *`
which means anything will get appended with
`    HostName %h.{{ dns.clusterid }}.{{ dns.domain }}`

I propose this be changed to a Jinja template...   and instead of *, just list the first part of the hostnames from the vars file.

**Comment by therevoman** [comment]
Adding this above the `Host *` entry helped some
```Host 192.168.88.*
    HostName %h
    User root
    IdentityFile /root/.ssh/id_rsa
    HostName %h```

my ipid is 192.168.88.0

**Comment by christianh814** [comment]
This is already fixed in the `devel` branch by a213ad53f40016393b1e590412c52d9f9df33eb3

Will merge into master sometime this week.
